libc/src/math/CMakeLists.txt

```
@@ -41,13 +41,14 @@ function(add_math_entrypoint_object name)
 endfunction()
 
 add_math_entrypoint_object(acos)
-add_math_entrypoint_object(acosf)
-add_math_entrypoint_object(acosf16)
-
 add_math_entrypoint_object(acosh)
+add_math_entrypoint_object(acosf)
 add_math_entrypoint_object(acoshf)
+add_math_entrypoint_object(acosf16)
 add_math_entrypoint_object(acoshf16)
 
+add_math_entrypoint_object(acospif16)
+ 
 add_math_entrypoint_object(asin)
 add_math_entrypoint_object(asinf)
 add_math_entrypoint_object(asinf16)
```

I meant putting `acospif16` between the `acosh*` and `asin*` functions, like this:

```suggestion
add_math_entrypoint_object(acos)
add_math_entrypoint_object(acosf)
add_math_entrypoint_object(acosf16)

add_math_entrypoint_object(acosh)
add_math_entrypoint_object(acoshf)
add_math_entrypoint_object(acoshf16)

add_math_entrypoint_object(acospif16)

add_math_entrypoint_object(asin)
add_math_entrypoint_object(asinf)
add_math_entrypoint_object(asinf16)
```

---

I think it should be good now